runtime/databricks/automl_runtime/forecast/__init__.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (C) 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+OFFSET_ALIAS_MAP = {
+    "W": "W",
+    "d": "D",
+    "D": "D",
+    "days": "D",
+    "day": "D",
+    "hours": "H",
+    "hour": "H",
+    "hr": "H",
+    "h": "H",
+    "H": "H",
+    "m": "min",
+    "minute": "min",
+    "min": "min",
+    "minutes": "min",
+    "T": "T",
```

Comment:
Just curious, what's 'T'?

Reply:
"T" is the same as "min". Probably we should keep it consistent to map to "min".

Reply:
Yes, I think it should map to `min`. Otherwise when you `_fill_missing_time_steps` and do `resample` with `T`, would it work?

Reply:
Replaced. BTW T does work with the pandas resample function.